### DISCUSSION TITLE
Electron build is not happening

### DISCUSSION THREAD

**Discussion by pranjali22pandey** [alex8088/electron-vite#622] [Q&A]
![Screenshot 2024-09-25 154112](https://github.com/user-attachments/assets/2ef90313-ac39-490d-8751-93242e4ffffe)
When I am using npm run dev my application is running fine , but when I using npm run build:win or npm start , my build process is working and build files are going out/renderer/assests but after that when my application is starting it is not coming fully just the nav bar is appearing and then showing errors like "Not allowed to load local resource: file:///C:/"  and "Failed to load URL: file:///C:/Users/PandeyPranja/vueElectronTest/6/out/renderer/ag-grid with error: ERR_FILE_NOT_FOUND" why is this happening 
![image](https://github.com/user-attachments/assets/45fbf243-dc7a-4c3e-b3ce-9dce63592d87)
![Screenshot 2024-09-25 155419](https://github.com/user-attachments/assets/2c2d6d2c-b583-4bfe-845b-a72f6c68462c)
this is how my application should come but npm start this is coming 
![Screenshot 2024-09-25 155632](https://github.com/user-attachments/assets/dc3e5162-cc54-4bee-ba10-cb931160b9df) what to do how to solve this ...... I have tried the things mentioned in documentation but it is not working 
![Screenshot 2024-09-25 164308](https://github.com/user-attachments/assets/97bf7b9b-ae28-476c-8d76-cb77a84d7bfb)
this is the error

**Comment by pranjali22pandey** [comment]
@tsmanuelanton @WolfAvenger  
@alex8088  could you please look into the matter

**Reply by tsmanuelanton** [reply]
I don’t work on Windows, but let’s give it a shot. Can you share the line of code where you're importing the asset? Maybe it’s the`/` before `C:` causing the issue.

**Reply by pranjali22pandey** [reply]
![image](https://github.com/user-attachments/assets/7a41b20f-adcd-430b-aaa7-969fc05ccfce)

**Reply by tsmanuelanton** [reply]
JS files must go  in folder `renderer/src` and you have to impot them in your html files from the folder `renderer` with the original filenames not the one set by vite in the out folder. Hope that works

**Comment by alex8088** [comment]
By default, the working directory of renderers are located in `src/renderer`, so the public directory needs to be created in this directory. The default public directory is named `public`.

**Reply by pranjali22pandey** [reply]
The issue is not with build but with electron-vite not working with routing , i changed the routing method now it is working but i have other doubt    
![image](https://github.com/user-attachments/assets/c6074102-6495-4643-90ca-96d8d3fb0a08)
   i  want that i should give the answers to below question in the command itself like add typescript answer as no and other also , how can i do that using commands only
